core-schema/src/main/resources/sql-schema/V8__Mod_clans.sql

```diff
@@ -87,3 +87,37 @@ CREATE FUNCTION clans_add_enemy
     INSERT INTO clans_clan_enemies (clan_id, enemy_id) VALUES (clan_identifier, enemy_identifier);
     RETURN 0;
   END;
+
+  -- A248, i know very little about sql but this is very interesting: Are you implementing an error-handler like catch
+  -- clause here? is 1062 the error code? If so, sql user defined functions are more incredible than i thought!
+  -- I wholly apologize for doubting these earlier in our discussion: we may use these to circumvent writing
+  -- methods with excessive querying. With these functions we can reduce all of our java-sql interactions to a single
+  -- function query: Below is a function that aims to reduce the amount of java-sql interactions of the Create Clan method.
```

Comment:
If by "here", you mean the code above this comment, then yes. I truly appreciate your forthrightness, but I must return the favor - I spent more time than necessary on converting parts of dataloader to use stored procedures than was necessary. Really, what we need to use stored routines for is one thing and only one thing - using error handlers for 1062 instead of INSERT IGNORE, because INSERT IGNORE has the potential for data corruption and we have to avoid that possibility entirely. As for other uses of stored routines, I agree that routines present a nice SQL interface with logic contained in a single function, and it's fine to use them for new code, but we do not need to change existing methods like createClan to use them. After all, createClan is not intended to be an often-called method. Worry about performance optimization in context.